### DISCUSSION TITLE
smallrye config with list support comma-separated profiles?

### DISCUSSION THREAD

**Discussion by agreedSkiing** [quarkusio/quarkus#38163] [Q&A] [answered]
Been experimenting for a while to no avail and so why not ask the question, does the smallrye config object/interface support comma-separated list of profile names?

https://quarkus.io/guides/config-reference#multiple-profiles

P.S. Quarkus project is using the 3.6.4 version.

P.P.S. The smallrye config interface seems to be aware of applicaiton-<profile>.properties files

P.P.P.S This seems to be an issue when using a list and %dev,test,something.app.* see my project https://github.com/agreedSkiing/reproducer-config-list-comma-separated-profiles

[a comment by an automated account is not shown]

**Reply by radcortez** [reply, marked as answer]
https://github.com/quarkusio/quarkus/pull/38027 will fix this.

I've already tried it with the reproducer, and all tests pass.
